views/show.tmpl

```diff
@@ -1,3 +1,24 @@
+{{define "title"}}{{.PageContext.ShortName}} | {{with .PageData}}{{.Show.Title}}{{end}}{{end}}
+
+{{define "open-graph"}}
+	<meta name="twitter:card" content="summary_large_image"/>
```

Comment:
@Brookke yeah that sounds a good suggestion. I think I will do that to shows too?